utils/logger/interface.go

```diff
@@ -78,3 +78,9 @@ func GetOrCreateGlobalLoggerEx(ctx context.Context, baseLevel LogLevel, stdOutFi
 func GetGlobalLogger() Logger {
 	return globalLogger
 }
+
+type Formatter interface {
```

Comment:
I was thinking maybe something less generic, like "LogFormatter" ? But it's fine anyway as there is  the 'logger' package/namespace.

Reply:
Following the go best practices, you need to avoid using the package name in the beginning of the type names.